pkg/dumper/mysql/dumper.go

```diff
@@ -0,0 +1,136 @@
+package mysql
+
+import (
+	"database/sql"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"strings"
+	"sync/atomic"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/hellofresh/klepto/pkg/database"
+	"github.com/hellofresh/klepto/pkg/dumper"
+	"github.com/hellofresh/klepto/pkg/dumper/generic"
+	"github.com/hellofresh/klepto/pkg/reader"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// myDumper dumps a database into a mysql db
+type myDumper struct {
+	conn   *sql.DB
+	reader reader.Reader
+}
+
+func NewDumper(conn *sql.DB, rdr reader.Reader) dumper.Dumper {
+	return generic.NewSqlDumper(
+		rdr,
+		&myDumper{
+			conn:   conn,
+			reader: rdr,
+		},
+	)
+}
+
+func (p *myDumper) DumpStructure(sql string) error {
+	if _, err := p.conn.Exec(sql); err != nil {
```

Comment:
Here we can just `return p.conn.Exec(sql)`

Reply:
Actual you cant since the exec return is `(Result, error)`

Reply:
True thing, I didn't notice the `_` 👍